### PR TITLE
Added single video page

### PR DESCRIPTION
**Pull request by muskanbagrecha**
In this PR, I have added the single video page using useParams hook.

No other functionality has been added as of now. 
Files to view: [files](https://github.com/muskanbagrecha/video-library-react-app/pull/5/files?file-filters%5B%5D=.jsx&show-viewed-files=true)

### CODE REVIEW

File: src/Pages/VideoPage/VideoPage.jsx [hunk 1]
```diff
@@ -1,18 +1,115 @@
...
-          thumbnail={thumbnail}
-          title={title}
-          creator={creator}
-          views={views}
-          date={date}
-        />
-      </div>
-      <div className="most-watched-container"></div>
+    <div className="sub-container video-page-container">
+      {/* {videoLoader && <img src={spinner} alt="loading.." />} */}
```
Comment (tarunsankhla) [inline src/Pages/VideoPage/VideoPage.jsx:46]: Remove commented code

File: src/Pages/VideoPage/VideoPage.jsx [hunk 2]
```diff
@@ -1,18 +1,115 @@
...
+    serverCall: fetchCurrentVideo,
+    error,
+  } = useFetch();
+
+  useEffect(() => {
+    if (currentVideoResponse === null) {
+      fetchCurrentVideo({ method: "get", url: `/api/video/${videoURLId}` });
+      console.log(error);
+    }
+    if (currentVideoResponse?.video._id !== videoURLId) {
```
Comment (tarunsankhla) [inline src/Pages/VideoPage/VideoPage.jsx:34]: You can use or operator and add both of them in same if loop

File: src/Pages/Homepage/Homepage.jsx [hunk 3]
```diff
@@ -3,29 +3,28 @@ import { Banner, CategorySection } from "./Components/";
...
 import spinner from "../../Assets/spinner.svg";
 
 export const Homepage = () => {
   const [loader, setLoader] = useState(true);
-  const { showAlert } = useAlert();
+  
   useEffect(() => {
     setTimeout(() => {
       setLoader(false);
     }, 500);
```
Comment (tarunsankhla) [inline src/Pages/Homepage/Homepage.jsx:16]: Why time out of 0.5sec you already have it in use effect

**Review by tarunsankhla — APPROVED** [review]
Everything looks good 
Minor changes and you can go ahead and merge it

File: src/Pages/VideoPage/VideoPage.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+.video-container::-webkit-scrollbar {
+    display: none;
+}
+
+.side-container {
+    grid-area: sidebar-container;
+    display: flex;
+    flex-direction: column;
+    /* justify-content: center; */
```
Comment (zoef-yt) [inline src/Pages/VideoPage/VideoPage.css:26]: remove commented code.

File: src/Pages/ExplorePage/ExploreFilter.jsx [hunk 5]
```diff
@@ -16,7 +16,6 @@ export const ExploreFilter = () => {
       try {
         const res = await axios.get("/api/categories");
         if (res.status === 200 || res.status === 201) {
```
Comment (zoef-yt) [inline src/Pages/ExplorePage/ExploreFilter.jsx:18]: This check for status can be removed. You can directly use setData inside of it.

File: src/index.css [hunk 6]
```diff
@@ -3,16 +3,34 @@
...
-:root{
-    --transparent-black: rgba(0,0,0,0.5);
-    --transparent-primary: rgba(145, 128, 196,0.5);
-    --box-shadow: 0 2px 8px rgba(0,0,0,0.25);
-    --container-box-shadow: 0 2px 8px rgba(0,0,0,0.25);
+body,
+html {
+    scroll-behavior: smooth;
+    letter-spacing: 0.25px;
+    /* font-family: 'Roboto', sans-serif; */
```
Comment (zoef-yt) [inline src/index.css:10]: remove commented code.

File: src/index.css [hunk 7]
```diff
@@ -57,62 +75,83 @@ footer{
...
     flex-wrap: wrap;
     justify-content: center;
     margin: 0 auto;
-    width: 80%;
+    width: 90%;
 }
 
-.card-container{
+.card-container {
     /* justify-content: start; */
```
Comment (zoef-yt) [inline src/index.css:93]: remove commented code.

**Review by saurabhchirde — APPROVED** [review]
Couldn't find any issue, as your code is clean and easy to read. 
I guess some code is commented on for the development purpose so I ignored it.
Regarding the UI, it's great and bolder, loved the landing page and the welcome screen. You can proceed to merge.

**Review by zoef-yt — APPROVED** [review]
The code is really great. The UI looks really clean and amazing. Great work Muskan you are good to merge.

**Pull request merged**